apps/server/src/shared/testing/factory/h5p-content.factory.ts

```diff
@@ -0,0 +1,26 @@
+import { ContentMetadata, H5PContent } from '@src/modules/h5p-editor/contentStorage/h5p-content.entity';
+import { BaseFactory } from './base.factory';
+
+class H5PContentFactory extends BaseFactory<H5PContent, { metadata: ContentMetadata; content: unknown }> {}
+
+export const h5pContentFactory = H5PContentFactory.define(H5PContent, ({ sequence }) => {
+	return {
+		content: {
+			[`field${sequence}`]: sequence,
+			dateField: new Date(sequence),
+			thisObjectHasNoStructure: true,
+			nested: {
+				works: true,
+			},
+		},
+		metadata: new ContentMetadata({
+			defaultLanguage: 'de-de',
+			embedTypes: ['iframe'],
+			language: 'de-de',
+			license: `License #${sequence}`,
+			mainLibrary: `License-${sequence}.0`,
```

Comment:
I don't have much clue what are reasonable values here, but should this maybe be "Library-${sequence}.0"?